code/model/modelread.cpp

```diff
@@ -1738,14 +1771,37 @@ int read_model_file(polymodel * pm, const char *filename, int n_subsystems, mode
 			}
 
 			case ID_SLDC: // kazan - Shield Collision tree
-				{
+				{   //ShivanSpS - if pof version is 2118 or higher ignore SLDC, otherwise convert it to slc2.
+					if (pm->version < 2118) {
+						mprintf(("SLDC data is being converted to SLC2.\n"));
```

Comment:
Same here, please remove or convert to `nprintf`.

Reply:
ok